.travis.yml

```diff
@@ -61,11 +70,10 @@ before_install:
   # NANOPB
   - git clone https://github.com/nanopb/nanopb.git ${NANOPB_PATH}
   - pushd ${NANOPB_PATH} && git checkout nanopb-${NANOPB_VERSION} && cd generator/proto && make && popd
+  # Newer Protobuf
+  - wget ${PROTO_REPO}/v${PROTO_VERSION}/${PROTO_ZIP}; sudo unzip ${PROTO_ZIP} -d ${PROTO_DIR}
```

Comment:
sudo?

Reply:
yap! cause we unzip to ${PROTO_DIR} = /usr/local/proto3 

maybe better location?

Reply:
This part of the main README.md will be very confusing ... in particular, we now need two different versions of protocol buffers.

1. protobuf 3.0.x to build the SGX SDK

1. protobuf 3.11.x or higher to build the fabric protos

However, the tricky part is that we build nanopb using the python bindings for 3.0.x ... argh ... need more testing to make this much easier.